zulipterminal/model.py

```diff
@@ -590,6 +595,23 @@ def update_rendered_view(self, msg_id: int) -> None:
                 self.msg_list.log[msg_pos] = new_msg_w
                 self.controller.update_screen()
 
+                if self.message_star_toggled:  # Updates next content_header
+                    try:
+                        next_msg_box = self.msg_list.log[pos].original_widget
+                        next_msg_w_list = create_msg_box_list(
+                                        self, [next_msg_box.message['id']],
+                                        last_message=next_msg_box.last_message)
+                        if not next_msg_w_list:
+                            return
+                        else:
+                            next_new_msg_w = next_msg_w_list[0]
+                        next_msg_pos = self.msg_list.log.index(
+                                            self.msg_list.log[pos])
+                        self.msg_list.log[next_msg_pos] = next_new_msg_w
+                        self.controller.update_screen()
+                    except:
```

Comment:
Yes, to make sure it doesn't crash in case the last message is starred.

Should I add that it's an IndexError? What else do you suggest to avoid this?